Pathfinding: use rolling stop maximum speed and length

Sh099078 opened the pull request:
Fixes #8154.

Sh099078 committed:
core, editoast, front: pathfinding: use rolling stop maximum speed and length

Erashin commented on core/src/main/kotlin/fr/sncf/osrd/api/api_v2/pathfinding/PathfindingBlocksEndpointV2.kt:
```diff
@@ -94,7 +93,7 @@ fun runPathfinding(
         )
 
     // TODO: add the rolling stock global speed limit to the request
```
Remove TODO, you just fixed it :)

Erashin reviewed (approved):
Core part looks good to me, gj :)

leovalais commented on editoast/src/views/v2/path/pathfinding.rs:
```diff
@@ -54,6 +55,12 @@ struct PathfindingInput {
     /// List of waypoints given to the pathfinding
     path_items: Vec<PathItemLocation>,
+    /// Rolling stock maximum speed
+    #[schema(value_type=f64)]
+    rolling_stock_maximum_speed: OrderedFloat<f64>,
+    /// Rolling stock length
+    #[schema(value_type=f64)]
```
```suggestion
    #[schema(value_type = f64)]
    rolling_stock_maximum_speed: OrderedFloat<f64>,
    /// Rolling stock length
    #[schema(value_type = f64)]
```

leovalais reviewed (approved):
LGTM, congratz! 🚀

Erashin reviewed (commented):
Some e2e tests are failing though, will need looking into.

Khoyo commented:
> Some e2e tests are failing though, will need looking into.

Yeah, looking at the videos they seem to have a really low timeout. We should consider increasing it at least until osrdyne is able to correctly report core readiness.

Sh099078 committed:
front: increase max timeout on expect in e2e tests

The previous maximum timeout of 5000ms was insufficient for core to return
the calculated pathfinding. Double the timeout to make the tests pass.

clarani reviewed (approved):
LGTM :white_check_mark: tested
